<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsComposerBase</class>
 <widget class="Q3MainWindow" name="QgsComposerBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>526</height>
   </rect>
  </property>
  <property name="mouseTracking" >
   <bool>false</bool>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Map Composer</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>38</y>
     <width>725</width>
     <height>488</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QSplitter" name="mSplitter" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy" >
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="lineWidth" >
       <number>1</number>
      </property>
      <property name="midLineWidth" >
       <number>0</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="Q3Frame" name="mViewFrame" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth" >
        <number>1</number>
       </property>
      </widget>
      <widget class="QTabWidget" name="mOptionsTabWidget" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>150</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>300</width>
         <height>32767</height>
        </size>
       </property>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>8</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="0" colspan="2" >
          <widget class="Q3Frame" name="mCompositionOptionsFrame" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>7</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth" >
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="textLabel1" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Composition</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="mCompositionNameComboBox" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>Item</string>
        </attribute>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>8</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="Q3Frame" name="mItemOptionsFrame" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>7</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth" >
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Q3ToolBar" name="Toolbar_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>223</width>
     <height>38</height>
    </rect>
   </property>
   <property name="label" >
    <string>Toolbar_2</string>
   </property>
   <addaction name="actionSelectItem" />
   <addaction name="actionAddMap" />
   <addaction name="actionAddVectorLegend" />
   <addaction name="actionAddScalebar" />
   <addaction name="actionAddLabel" />
   <addaction name="actionAddPicture" />
  </widget>
  <widget class="Q3ToolBar" name="Toolbar" >
   <property name="geometry" >
    <rect>
     <x>223</x>
     <y>0</y>
     <width>155</width>
     <height>38</height>
    </rect>
   </property>
   <property name="label" >
    <string>Toolbar</string>
   </property>
   <addaction name="actionZoomIn" />
   <addaction name="actionZoomOut" />
   <addaction name="actionZoomFullExtent" />
   <addaction name="actionRefresh" />
  </widget>
  <widget class="Q3ToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>378</x>
     <y>0</y>
     <width>121</width>
     <height>38</height>
    </rect>
   </property>
   <property name="label" >
    <string>Tools</string>
   </property>
   <addaction name="actionPrint" />
   <addaction name="actionImage" />
   <addaction name="actionSvg" />
  </widget>
  <action name="fileOpenAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>themes/default/mActionFileOpen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open Template ...</string>
   </property>
   <property name="iconText" >
    <string>Open Template</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileSaveAsAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>themes/default/mActionFileSave.png</iconset>
   </property>
   <property name="text" >
    <string>Save Template &amp;As...</string>
   </property>
   <property name="iconText" >
    <string>Save Template As</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="icon" >
    <iconset>themes/default/mActionFilePrint.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Print...</string>
   </property>
   <property name="iconText" >
    <string>Print</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionZoomFullExtent" >
   <property name="icon" >
    <iconset>themes/default/mActionZoomFullExtent.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom All</string>
   </property>
   <property name="iconText" >
    <string>Zoom to full extent</string>
   </property>
  </action>
  <action name="actionZoomIn" >
   <property name="icon" >
    <iconset>themes/default/mActionZoomIn.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom In</string>
   </property>
   <property name="iconText" >
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut" >
   <property name="icon" >
    <iconset>themes/default/mActionZoomOut.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Out</string>
   </property>
   <property name="iconText" >
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionAddMap" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>themes/default/mActionAddNonDbLayer.png</iconset>
   </property>
   <property name="text" >
    <string>Add new map</string>
   </property>
   <property name="iconText" >
    <string>Add new map</string>
   </property>
  </action>
  <action name="actionAddLabel" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>themes/default/mActionLabel.png</iconset>
   </property>
   <property name="text" >
    <string>Add new label</string>
   </property>
   <property name="iconText" >
    <string>Add new label</string>
   </property>
  </action>
  <action name="actionAddVectorLegend" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>themes/default/mActionAddLegend.png</iconset>
   </property>
   <property name="text" >
    <string>Add new vect legend</string>
   </property>
   <property name="iconText" >
    <string>Add new vect legend</string>
   </property>
  </action>
  <action name="actionSelectItem" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>themes/default/mActionSelect.png</iconset>
   </property>
   <property name="text" >
    <string>Select/Move item</string>
   </property>
   <property name="iconText" >
    <string>Select/Move item</string>
   </property>
  </action>
  <action name="actionImage" >
   <property name="icon" >
    <iconset>themes/default/mActionExportMapServer.png</iconset>
   </property>
   <property name="text" >
    <string>Export as image</string>
   </property>
   <property name="iconText" >
    <string>Export as image</string>
   </property>
  </action>
  <action name="actionSvg" >
   <property name="icon" >
    <iconset>themes/default/mActionSaveAsSVG.png</iconset>
   </property>
   <property name="text" >
    <string>Export as SVG</string>
   </property>
   <property name="iconText" >
    <string>Export as SVG</string>
   </property>
  </action>
  <action name="actionAddScalebar" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>themes/default/mActionMeasure.png</iconset>
   </property>
   <property name="text" >
    <string>Add new scalebar</string>
   </property>
   <property name="iconText" >
    <string>Add new scalebar</string>
   </property>
  </action>
  <action name="actionRefresh" >
   <property name="icon" >
    <iconset>themes/default/mActionDraw.png</iconset>
   </property>
   <property name="text" >
    <string>Refresh view</string>
   </property>
   <property name="iconText" >
    <string>Refresh view</string>
   </property>
  </action>
  <action name="actionAddPicture" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>themes/default/mActionAddRasterLayer.png</iconset>
   </property>
   <property name="iconText" >
    <string>Add new picture</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends></extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3ToolBar</class>
   <extends></extends>
   <header>q3listview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends></extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
